docs/docsite/rst/scenario_guides/guide_infoblox.rst

```diff
@@ -243,50 +243,55 @@ To configure a reverse DNS zone:
             state: present
             provider: "{{ nios_provider }}"
 
-Dynamic inventory script
+Dynamic inventory plugin
 ========================
 
-You can use the Infoblox dynamic inventory script to import your network node inventory with Infoblox NIOS. To gather the inventory from Infoblox, you need two files:
+Ansible provides a dynamic inventory plugin for Infoblox.
```

Comment:
Ansible does not do that. The Infoblox collection does. As long as infoblox.nios_modules is not included in Ansible, this is an important distinction :)